README.md

```diff
@@ -57,6 +57,16 @@ Runs tests and verifies against previously-recorded golden values. Failures gene
 
 For more examples, check out the [sample][sample] project.
 
+> **Note**: If running on JDK 16+, you must add the following two JVM args to your `Test` task.
```

Comment:
note this is using github's special formatting for this

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/1361086/188327769-dd332782-6438-4b36-8a22-e5fa0146f010.png">